cli.js

```diff
@@ -32,6 +32,11 @@ if (args.i && args._[0] === '-') {
   process.exit(1);
 }
 
+if (args.maxdepth && isNaN(parseInt(args.maxdepth))) {
```

Comment:
I don't think the `isNaN` check is needed, since minimist already casts ["numeric-looking arguments"](https://github.com/substack/minimist#var-argv--parseargsargs-opts) to strings.

But this gets me thinking. we should probably just pass `args` to `toc()` as the second argument, so any options can be defined on the command line. I'm not sure why we didn't do this before. 

thoughts?